modules/add_credit/src/AddCreditService.php

```diff
@@ -161,10 +162,30 @@ public function formCommerceProductTypeEditFormAlter(&$form, FormStateInterface
       '#title' => t('Enable <em>Apigee Monetization Add Credit</em> for this product type.'),
       '#default_value' => $default_value,
     ];
+
+    /** @var \Drupal\commerce_product\Entity\ProductTypeInterface $product_type */
+    $default_value = (($product_type = $form_state->getFormObject()->getEntity())
```

Comment:
`$product_type` is already defined on line 155.

Maybe change to something like: 

```
    $product_type = $form_state->getFormObject()->getEntity();

    // Add an option to allow enabling Apigee add credit for a product type.
    $form['apigee_m10n_enable_add_credit'] = [
      '#type' => 'checkbox',
      '#title' => t('Enable <em>Apigee Monetization Add Credit</em> for this product type.'),
      '#default_value' => $product_type->getThirdPartySetting('apigee_m10n_add_credit', 'apigee_m10n_enable_add_credit'),
    ];

```

etc...